### PR TITLE
updated careers md, pulled in copy from notion

### PR DESCRIPTION
**Pull request by 0xRowdy**

### CODE REVIEW

File: careers.md [hunk 1]
```diff
@@ -1 +1,16 @@
-Please see [notion](https://www.notion.so/shapeshift/e18d129303384d85987c047d0d1704de?v=c5607c51fb9a4f2fa1b214e4c426a650) for open roles.
+# Careers
+There are many ways that developers can get going in the ShapeShift DAO ecosystem and with our codebase.
+
+## Opportunities to Contribute
+- Join our standups at 10AM MST Mon-Fri in the StandUp-AllAccess channel on our [Discord](https://discord.gg/shapeshift) server
```
Comment (0xApotheosis) [inline careers.md:5]:
```suggestion
- Join our standups at 10AM MST Mon-Fri in the `StandUp-AllAccess` channel on our [Discord](https://discord.gg/shapeshift) server
```

File: careers.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+     - You first need to pass the Captcha-bot, to make sure you are a real person 
+     - Once you are on the server and see a bunch of channels, go to the #role-selector channel, and click on the icon for the "Engineering" role
```
Comment (0xApotheosis) [inline careers.md:7]:
```suggestion
     - Once you are on the server and see a bunch of channels, go to the `#role-selector` channel, and click on the icon for the "Engineering" role
```

File: careers.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+     - Then you will see the #StandUp-AllAccess channel, #engineering-public channel and many others
```
Comment (0xApotheosis) [inline careers.md:8]:
```suggestion
     - Then you will see the `#StandUp-AllAccess` channel, `#engineering-public` channel and many others
```

File: careers.md [hunk 4]
```diff
@@ -1 +1,16 @@
...
+- Check out Engineering Demos on the 1st and 3rd Wednesday of every month on Discord, on the ShapeShift Stage
+- Join the conversation and ask questions in the engineering-public channel on Discord
```
Comment (0xApotheosis) [inline careers.md:10]:
```suggestion
- Join the conversation and ask questions in the `#engineering-public` channel on Discord
```

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
A few suggestions. Also, I know it's not explicitly noted anywhere that we do it this way  but for markdown docs, could you please add a new line after each heading?

**Comment by 0xRowdy** [comment]
> A few suggestions. Also, I know it's not explicitly noted anywhere that we do it this way but for markdown docs, could you please add a new line after each heading?

Absolutely, I wanted to get some eyes on this PR before I start submitting others so I can get all of these kinds of comments resolved before submitting others. Thanks for the feedback!

**Review by 0xApotheosis — APPROVED** [review]

**Comment by 0xApotheosis** [comment]
Nice one, thanks for making our docs cleaner!

**Pull request merged**